Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+ruby '2.5.6'
+
+gem 'sorbet', :group => :development
+gem 'sorbet-runtime'
```

Comment:
We don't need to depend on sorbet-runtime.

Reply:
Removed sorbet-runtime gem in below commit.